### PR TITLE
fix: Album list image crop fix

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves [MOPLAT-411]

|Before|After|
|---|---|
|image|image|

### PR Checklist

- [ ] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-411]: https://artsyproduct.atlassian.net/browse/MOPLAT-411?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/sharedUI/items/AlbumListItem.tsx [hunk 1]
```diff
@@ -41,6 +29,7 @@ export const AlbumListItem = ({ album }: { album: Album }) => {
           {album.artworkIds.length} Artworks
         </Text>
       </Flex>
+      <Spacer mb={1} />
```
Comment (pvinis) [inline src/app/sharedUI/items/AlbumListItem.tsx:32]: non blocking: you can start using y instead of mb 😉

**Review by pvinis — APPROVED** [review]

File: src/app/sharedUI/items/AlbumListItem.tsx (hunk 1 above)
Comment (rajsam003) [reply src/app/sharedUI/items/AlbumListItem.tsx:32]: Ahh cool. Didn't know.

**Comment by admbtlr** [comment]
Can you add a screenshot of how this looks on iPad?

**Comment by rajsam003** [comment]
> Can you add a screenshot of how this looks on iPad?

Before

https://user-images.githubusercontent.com/17421923/189168105-2a06eb56-805f-4177-8a9e-c256c8c6eea1.mov



After

https://user-images.githubusercontent.com/17421923/189168152-e26a61d9-53d7-48a0-8fc1-d143d63e2053.mov

**Comment by admbtlr** [comment]
I think we need to have two columns for albums on mobile, but that will be a different ticket, so this is fine from my perspective

**Review by admbtlr — APPROVED** [review]

**Review by patrinoua — APPROVED** [review]

**Pull request merged**